.eslintrc

```diff
@@ -5,12 +5,10 @@
   },
   "extends": "eslint:recommended",
   "parserOptions": {
-    "ecmaVersion": 6
+    "ecmaVersion": 8
   },
   "rules": {
-    "curly": "error",
```

Comment:
IMO these rules should stay consistent across Medic code unless there is a significant reason not to do so

Reply:
I'd like to do so in a second PR is that is okay. The change became very large.  Tracking via https://github.com/medic/medic-conf/issues/137